app/src/main/java/com/fstyle/androidtrainning/screen/splash/SplashActivity.java

```diff
@@ -22,6 +23,7 @@
     public static final int REQUEST_PERMISSION_CODE = 101;
     public static final int TIME_HANDLER = 3000;
     public static final int SDK_DEFAULT = 23;
+    private String TAG = "chuong";
```

Comment:
WARNING: [SuspiciousConstantFieldName] The field name indicates a constant but its modifiers do not
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#SuspiciousConstantFieldName